CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ Try to use the following format:
 ### Changed
 ### Fixed
 
+## [NG.NG.NG]
```

Comment:
Don't forget to update this

Reply:
it will be updated via my bumpversion.sh script